tnb/banks.py

```diff
@@ -30,6 +30,40 @@ def fetch_validators(self):
         """
         return self.fetch("/validators")
 
+    def fetch_validator_confirmation_services(self) -> list:
```

Comment:
How do you feel about this syntax adding these sorts of annotations? I think either we should add it everywhere or avoid it just to be consistent. Thoughts?

Reply:
It is good to add these to remove ambiguity. For example on the following code it will force the user to use a string for the 
`msg_end` and `msg_start` parameters. 

```python
    def create_validator_confirmation_service(
            self, msg_end: str, msg_start: str, node_id: str,
            signature: str) -> dict:
```
These could easily be confused with `datetime` or `date` objects.

Reply:
I think let's remove these annotations for now. and create a new issue to move all the functions to annotations as well at once so we don't have different styles at one point in time. what do you guys think?
@CirXe0N @sbalasa @buckyroberts

Reply:
@shahraizali  @sbalasa 
Ok, no problem. I will create a new issue for the typing annotations.

Reply:
Here is the issue: https://github.com/thenewboston-developers/thenewboston-python-client/issues/39